### PR TITLE
course nodes

### PR DESCRIPTION
**Pull request by Yongjun-Kim-YJ**
added courses on the graph

### CODE REVIEW

File: src/main/webapp/homepage.jsp [hunk 1]
```diff
@@ -42,29 +42,6 @@
...
-<!-- 					        border-gray-300  -->
-<!-- 					        rounded-lg  -->
-<!-- 					        focus:outline-none  -->
-<!-- 					        focus:ring-2  -->
-<!-- 					        focus:ring-blue-500  -->
-<!-- 					        focus:border-blue-500  -->
-<!-- 					        text-gray-700  -->
-<!-- 					        placeholder-gray-400  -->
-<!-- 					        shadow-sm -->
-<!-- 					    "> -->
```
Comment (angelahuangg) [inline src/main/webapp/homepage.jsp:67]: are these lines supposed to be deleted?

Comment (danknessdra) [reply src/main/webapp/homepage.jsp:67]: Yeah, they are just a comment

Comment (Yongjun-Kim-YJ) [reply src/main/webapp/homepage.jsp:67]: oh my bad I deleted this part accidently

**Review by danknessdra — APPROVED** [review]
Looks good, but maybe we should fetch the prereqs from the database instead of hardcoded

File: src/main/webapp/homepage.jsp (hunk 1 above)
Comment (Yongjun-Kim-YJ) [reply src/main/webapp/homepage.jsp:67]: alright then that's fine

**Comment by Yongjun-Kim-YJ** [comment]
> Looks good, but maybe we should fetch the prereqs from the database instead of hardcoded

I found an error when the edge ("CS160", "CS100W") included the whole graph doesn't appear.

**Comment by danknessdra** [comment]
That's fine then! We can figure it out later

**Pull request merged**